app/routes/start/hello.js

```diff
@@ -4,6 +4,21 @@ import OnboardingRouteMixin from '../../mixins/onboarding-route';
 export default Ember.Route.extend(OnboardingRouteMixin, {
   currentUser: Ember.inject.service(),
 
+  actions: {
+ /**
+ * Action that transitions to next route when user hits `enter` key
+ * if firstName and lastName fields are populated
+ * 
+ * @method
+ */
+   attemptToContinue() {
```

Comment:
I think the `controllers/start/hello.js` would be a better place for this function. That way you won't have to call `this.controller` on anything. If you move
```
actions: {
    attemptToContinue() {
      let userHasName = !this.get('usersNameIsEmpty');

      if (userHasName) {
        this.send('continue');
      }
    },
  },
```
it would work.

Also, if you could align the documentation and add the method name that would be 👌 
ex.
```
/**
 * Description
 *
 * @method attemptToContinue
 */
attempToContinue() {
```

Here's the doc syntax for reference: http://yui.github.io/yuidoc/syntax/index.html